### PR TITLE
Temporarily increase master-size to 8vCPU in scalability 100 node jobs

### PR DESCRIPTION
**Pull request by mm4tt**
Ref. https://github.com/kubernetes/kubernetes/issues/94098

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 1]
```diff
@@ -31,6 +31,8 @@ presubmits:
         - --extract=local
         - --flush-mem-after-build=true
         - --gcp-nodes=100
+        # TODO(k/k#94098): Remove the line below once we get enough data.
+        - --gcp-master-size=n1-standard-8
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:35]: Let's do the same for post submit in master branch.

Comment (mm4tt) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:35]: Done

**Comment by wojtek-t** [comment]
/lgtm
/approve

Thanks

**Pull request merged**